src/frontend/barnetilsyn/steg/4-dine-barn/DineBarn.tsx

```diff
@@ -18,9 +18,11 @@ const DineBarn = () => {
                 <Label spacing>
                     <LocaleTekst tekst={dineBarnTekster.hvilke_barn_spm} />
                 </Label>
-                {person.barn.map((barn) => (
+                {person.barn.map((barn, indeks) => (
                     <Checkbox
-                        value={barn.skalHaBarnepass}
+                        key={indeks}
```

Comment:
I tilfelle rekkefølgen på barn i listen endrer seg
```suggestion
                        key={barn.id}
```